### PR TITLE
Made edits to the content based on feedback

### PR DESCRIPTION
**Pull request by priyaayyar**
Made fixes as per the discussion with Javier.

### CODE REVIEW

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc [hunk 1]
```diff
@@ -6,75 +6,75 @@ endif::[]
 
 API Manager supports several policies for injecting or removing HTTP headers from a request or response. The policies take effect before sending the request or response.
 
-The add request header policy injects one or more headers into a request from the client to the backend service, or overrides the value of an existing header.
-The remove request header policy prevents receipt of one or more specified headers sent from the client to the backend service.
+Add Request Header policy injects one or more headers into a request from the client to the backend service, or overrides the value of an existing header.
```
Comment (JaatadiaMulesoft) [inline modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]: maybe we can merge line 9 with 12. In both lines (9 and 12), the policy does not override the values, it adds new ones, the result will be a request with the old values plus the new ones.
E.g: Request with header "foo=value1" if we add the header "foo=value2" with the policy, the result will be a request with header "foo=value1, value2"

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc [hunk 2]
```diff
@@ -6,75 +6,75 @@ endif::[]
...
+The Remove Request Header policy prevents receipt of one or more specified headers sent from the client to the backend service.
```
Comment (JaatadiaMulesoft) [inline modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]: maybe we can merge line 10 and 13

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc [hunk 3]
```diff
@@ -6,75 +6,75 @@ endif::[]
 
 API Manager supports several policies for injecting or removing HTTP headers from a request or response. The policies take effect before sending the request or response.
```
Comment (JaatadiaMulesoft) [inline modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]: There are only two policies now, 
image
image

image

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc [hunk 4]
```diff
@@ -6,75 +6,75 @@ endif::[]
...
+//. In Policy Configuration, choose the XML file, for example, `add-request.xml`, that you downloaded.
+//. Click Add.
+//+
+//image::gw-add-custom-pol.png[]
+//+
+//The policy, add request header policy in this example, appears on the Custom Policies page.
+//+
+//image::gw-custom-pol-list.png[]
 
 === Applying the Add Request/Response Header Policies
```
Comment (JaatadiaMulesoft) [inline modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]: change to the new policy name "Header Injection Policy"

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc [hunk 5]
```diff
@@ -118,16 +118,16 @@ Perform the steps in this procedure to apply either of the following policies:
```
Comment (JaatadiaMulesoft) [inline modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]: change to the new policy name "Header Removal Policy"

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc [hunk 6]
```diff
@@ -118,16 +118,16 @@ Perform the steps in this procedure to apply either of the following policies:
 
 To apply the remove request header or remove response header policies:
```
Comment (JaatadiaMulesoft) [inline modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]: We only one policy for this two cases

**Review by JaatadiaMulesoft — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc (hunk 2 above)
Comment (priyaayyar) [reply modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]: Done! Thanks!

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc (hunk 3 above)
Comment (priyaayyar) [reply modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]: Changed! Thanks!

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc (hunk 4 above)
Comment (priyaayyar) [reply modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]: Done! Thanks!

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc (hunk 5 above)
Comment (priyaayyar) [reply modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]: Done! Thanks!

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc (hunk 6 above)
Comment (priyaayyar) [reply modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]: Ok changed!

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc [hunk 7]
```diff
@@ -4,79 +4,83 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: header, policy
 
-API Manager supports several policies for injecting or removing HTTP headers from a request or response. The policies take effect before sending the request or response.
+API Manager supports policies for injecting or removing HTTP headers from a request or response. The policies take effect before sending the request or response.
```
Comment (kbostrom) [inline modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]: "injecting or removing": I would think a more parallel phrase would be "inserting or removing" (or the pair "injecting" and "ejecting," although I think that sounds odd...).

Comment (kbostrom) [reply modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]: Further, if the subsequent title uses "add," then maybe we want "adding or removing."

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc [hunk 8]
```diff
@@ -4,79 +4,83 @@ include::_attributes.adoc[]
...
 
-The add request header policy injects one or more headers into a request from the client to the backend service, or overrides the value of an existing header.
-The remove request header policy prevents receipt of one or more specified headers sent from the client to the backend service.
+== Add Request and Add Response Headers
 
-The add response header policy injects one or more headers into a response from the backend service to the client, or overrides the header value in a response from the backend service.
-The remove response header policy prevents inclusion of one or more specified headers in a response from the backend service to the client.
+The Add Request Header policy injects one or more headers into a request from the client to the backend service, resulting in a request with old values and new ones. The Add Response Header policy injects one or more headers into a response from the backend service to the client.
```
Comment (kbostrom) [inline modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]: ditto re. "inject"

Comment (kbostrom) [reply modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]: I don't understand how the first part of the first sentence logically leads to "resulting in a request with old values and new ones." Should we say "injects one or more new headers"? And if the response heading similarly injects new headers, why does it not result in old and new as well?

If this is something obvious to your audience (just not to me), then ignore.

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc [hunk 9]
```diff
@@ -4,79 +4,83 @@ include::_attributes.adoc[]
...
-* Add Response Headers Policy
-+
-`+https://www.anypoint.mulesoft.com/exchange/?search=Add%20Response%20Headers%20Policy+`
-+
-* Remove Response Headers Policy
-+
-`+https://www.anypoint.mulesoft.com/exchange/?search=Remove%20Response%20Headers%20Policy+`
-+
-. Unzip each policy archive.
+For example, in the a request with header "foo=value1", if we add the header "foo=value2" with the policy, the result will be a request with header "foo=value1, value2"
```
Comment (kbostrom) [inline modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]:
```suggestion
For example, in a request with the header "foo=value1", if you add the header "foo=value2" to the policy, the result is a request with the header "foo=value1, value2".
```

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc [hunk 10]
```diff
@@ -4,79 +4,83 @@ include::_attributes.adoc[]
...
-. In Policy Definition, click Choose File to locate the YAML file, `add-request.yaml` in this example, that you downloaded.
-. In Policy Configuration, choose the XML file, for example, `add-request.xml`, that you downloaded.
-. Click Add.
-+
-image::gw-add-custom-pol.png[]
-+
-The policy, add request header policy in this example, appears on the Custom Policies page.
-+
-image::gw-custom-pol-list.png[]
+* Assume the role of Organization Administrators or API Versions Owner for an API version you want to manage in Anypoint Platform. Applying policies is an API version management task.
```
Comment (kbostrom) [inline modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]:
```suggestion
* Suppose that you are an organization administrator or an API version owner for an API version you want to manage in Anypoint Platform. One of your API version management tasks is applying policies.
```

Comment (kbostrom) [inline modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]: To what text introduction do these list items apply?

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc [hunk 11]
```diff
@@ -4,79 +4,83 @@ include::_attributes.adoc[]
...
+* Ensure that you are licensed to use API Manager functionality (Mule 3.8.x runtime) or API Gateway runtime 2.x.
```
Comment (kbostrom) [inline modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]: "runtime" here refers to the runtime itself and not the runtime engine, right?

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc [hunk 12]
```diff
@@ -4,79 +4,83 @@ include::_attributes.adoc[]
...
 
-=== Applying the Add Request/Response Header Policies
+// == Downloading Add or Remove Header Policies
+
+//Currently, the add or remove header policies are available on Anypoint Exchange. Follow the step-by-step procedures to download, set up, and apply these policies.
+
+//To download the header policies:
+
+//. Login to Anypoint Platform and go to Exchange.
```
Comment (kbostrom) [inline modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]:
```suggestion
//. Log in to Anypoint Platform and navigate to Anypoint Exchange.
```

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc [hunk 13]
```diff
@@ -4,79 +4,83 @@ include::_attributes.adoc[]
...
+//+
+//`+https://www.anypoint.mulesoft.com/exchange/?search=Add%20Response%20Headers%20Policy+`
+//+
+//* Remove Response Headers Policy
+//+
+//`+https://www.anypoint.mulesoft.com/exchange/?search=Remove%20Response%20Headers%20Policy+`
+//+
+//. Unzip each policy archive.
+
+//An XML and YAML file appears in each of the folders created by unzipping the archives. For example, the add-request-header-policy folder contains the following files:
```
Comment (kbostrom) [inline modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]: Both an XML and a YAML

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc [hunk 14]
```diff
@@ -4,79 +4,83 @@ include::_attributes.adoc[]
...
+
+//* add-request.xml
+//* add-request.yaml
+
+//You use these files to set up the policy as described later.
```
Comment (kbostrom) [inline modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]:
```suggestion
//You use these files to set up the policy.
```

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc [hunk 15]
```diff
@@ -4,79 +4,83 @@ include::_attributes.adoc[]
...
+
+
+//== Setting Up and Applying Header Policies
+
+//If you have configured custom policies in Anypoint Platform, the process is basically the same for add or remove header policies. The following steps show how to set //up and apply the policies in Anypoint Platform.
```
Comment (kbostrom) [inline modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]:
```suggestion
//If you have configured custom policies in Anypoint Platform, the process is basically the same for adding or removing header policies. The following steps show how to set up and apply the policies in Anypoint Platform.
```

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc [hunk 16]
```diff
@@ -4,79 +4,83 @@ include::_attributes.adoc[]
...
+
+//To set up header policies:
+
+//. Go to the main API Administration page.
+//+
+//. On the main menu drop-down, select Custom policies. (Top right corner)
```
Comment (kbostrom) [inline modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]:
```suggestion
//. In the main menu at the top right, select *Custom policies*.
```

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc [hunk 17]
```diff
@@ -4,79 +4,83 @@ include::_attributes.adoc[]
...
+//+
+//Custom Policies appears listing custom policies, if there are any, that are available for the API.
```
Comment (kbostrom) [inline modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]: omit. This is the logical result of the previous step, which the user sees onscreen.

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc [hunk 18]
```diff
@@ -4,79 +4,83 @@ include::_attributes.adoc[]
...
+//+
+//. Click Add Custom Policy.
```
Comment (kbostrom) [inline modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]:
```suggestion
//. Click *Add Custom Policy*.
```

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc [hunk 19]
```diff
@@ -4,79 +4,83 @@ include::_attributes.adoc[]
...
+//. In Name, enter a name for the policy, for example `add request header policy`.
```
Comment (kbostrom) [inline modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]:
```suggestion
//. In Name, enter a name for the policy: for example, `add request header policy`.
```

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc [hunk 20]
```diff
@@ -4,79 +4,83 @@ include::_attributes.adoc[]
...
+//. In Policy Definition, click Choose File to locate the YAML file, `add-request.yaml` in this example, that you downloaded.
```
Comment (kbostrom) [inline modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]:
```suggestion
//. In Policy Definition, click *Choose File* to locate the YAML file that you downloaded (`add-request.yaml`).
```

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc [hunk 21]
```diff
@@ -4,79 +4,83 @@ include::_attributes.adoc[]
...
+//. In Policy Configuration, choose the XML file, for example, `add-request.xml`, that you downloaded.
```
Comment (kbostrom) [inline modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]:
```suggestion
//. In Policy Configuration, choose the XML file that you downloaded: for example, `add-request.xml`.
```

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc [hunk 22]
```diff
@@ -4,79 +4,83 @@ include::_attributes.adoc[]
...
+//. Click Add.
```
Comment (kbostrom) [inline modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]:
```suggestion
//. Click *Add*.
```

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc [hunk 23]
```diff
@@ -4,79 +4,83 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: header, policy
 
-API Manager supports several policies for injecting or removing HTTP headers from a request or response. The policies take effect before sending the request or response.
+API Manager supports policies for injecting or removing HTTP headers from a request or response. The policies take effect before sending the request or response.
 
-The add request header policy injects one or more headers into a request from the client to the backend service, or overrides the value of an existing header.
-The remove request header policy prevents receipt of one or more specified headers sent from the client to the backend service.
+== Add Request and Add Response Headers
```
Comment (JaatadiaMulesoft) [inline modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]: It should be "Header Injection Policy" (here and in the text)

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc (hunk 22 above)
Comment (kbostrom) [reply modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]: HAD TO END EDIT HERE DUE TO MEETING; please hold until noon if you can?

**Review by kbostrom — APPROVED** [review]
See final comment and let me know in Slack if I can finish.

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc [hunk 24]
```diff
@@ -111,23 +115,8 @@ The policy you applied moves from Available Policies to the Applied Policies lis
 
 === Applying the Remove Request/Response Header Policies
```
Comment (JaatadiaMulesoft) [inline modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]: It should be "Header Removal policy" (here and in the text)

**Review by JaatadiaMulesoft — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc [hunk 25]
```diff
@@ -4,79 +4,83 @@ include::_attributes.adoc[]
...
+//+
+//. Click Add Custom Policy.
+//. In Name, enter a name for the policy, for example `add request header policy`.
+//. In Policy Definition, click Choose File to locate the YAML file, `add-request.yaml` in this example, that you downloaded.
+//. In Policy Configuration, choose the XML file, for example, `add-request.xml`, that you downloaded.
+//. Click Add.
+//+
+//image::gw-add-custom-pol.png[]
+//+
+//The policy, add request header policy in this example, appears on the Custom Policies page.
```
Comment (kbostrom) [inline modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]:
```suggestion
//The new policy appears on the Custom Policies page.
```

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc [hunk 26]
```diff
@@ -4,79 +4,83 @@ include::_attributes.adoc[]
...
+//+
+//image::gw-custom-pol-list.png[]
+
+=== Applying the Header Injection Policies
 
 Continuing with example of the add request header policy, follow steps in this procedure to apply either of the following policies:
```
Comment (kbostrom) [inline modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]:
```suggestion
Continuing with the example of the Add Request Header policy, follow the steps in this procedure to apply either of the following policies:
```

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc [hunk 27]
```diff
@@ -85,17 +89,17 @@ Continuing with example of the add request header policy, follow steps in this p
 
 To apply add the request header policies or add response header policies:
```
Comment (kbostrom) [inline modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]: Init. cap. policy names, but leave "policy" lowercase. GLOBAL, per Term Presentation List.

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc [hunk 28]
```diff
@@ -85,17 +89,17 @@ Continuing with example of the add request header policy, follow steps in this p
...
-The add request header policy, for example, appears in Available Policies.
-+
-If the header was already provided by the client or another policy, the value is  overwritten by the one you provide in this policy.
-+
-. Click Apply.
-+
-The Apply add request header policy dialog box appears.
-+
+//. Assuming you already deployed the API version, select the version on the API Administration page.
+//. On the API version details page, select Policies from the list of items.
```
Comment (kbostrom) [inline modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]:
```suggestion
//. On the API version details page, select *Policies*.
```

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc [hunk 29]
```diff
@@ -85,17 +89,17 @@ Continuing with example of the add request header policy, follow steps in this p
...
+//+
+//The add request header policy, for example, appears in Available Policies.
```
Comment (kbostrom) [inline modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]: Suggest omitting as the logical result of the previous step and something evident onscreen to user.

Comment (kbostrom) [reply modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]: If you keep, init. cap. policy name.

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc [hunk 30]
```diff
@@ -85,17 +89,17 @@ Continuing with example of the add request header policy, follow steps in this p
...
+//+
+//If the header was already provided by the client or another policy, the value is  overwritten by the one you provide in this policy.
```
Comment (kbostrom) [inline modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]: How is this related to the step it follows? Do we really have to document what they can already see in the UI?

Comment (kbostrom) [reply modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]:
```suggestion
//If a header was provided by the client or another policy, it is overwritten by the one you provide in this policy.
```

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc [hunk 31]
```diff
@@ -85,17 +89,17 @@ Continuing with example of the add request header policy, follow steps in this p
...
+//+
+//. Click Apply.
```
Comment (kbostrom) [inline modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]:
```suggestion
//. Click *Apply*.
```

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc [hunk 32]
```diff
@@ -85,17 +89,17 @@ Continuing with example of the add request header policy, follow steps in this p
...
+//+
+//The Apply add request header policy dialog box appears.
```
Comment (kbostrom) [inline modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]: Suggest omitting as self-evident.

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc [hunk 33]
```diff
@@ -85,17 +89,17 @@ Continuing with example of the add request header policy, follow steps in this p
...
+//+
 . Type the name and value of a header. For example, type `extra-header` on the left and `myvalue` on the right.
```
Comment (kbostrom) [inline modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]:
```suggestion
. Type the name and value of a header. 
For example, type `extra-header` on the left and `myvalue` on the right. Optionally, include MEL expressions in the value.
```

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc [hunk 34]
```diff
@@ -85,17 +89,17 @@ Continuing with example of the add request header policy, follow steps in this p
...
 +
 . Optionally, include MEL expressions in the value of the header.
```
Comment (kbostrom) [inline modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]: omit

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc [hunk 35]
```diff
@@ -111,23 +115,8 @@ The policy you applied moves from Available Policies to the Applied Policies lis
 
 === Applying the Remove Request/Response Header Policies
 
-Perform the steps in this procedure to apply either of the following policies:
+To apply the Header Removal Policy:
```
Comment (kbostrom) [inline modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]:
```suggestion
To apply the Header Removal policy:
```

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc [hunk 36]
```diff
@@ -136,43 +125,43 @@ You can use wildcards to remove properties that have similar names. For example,
 +
 The policy you applied moves from the Available policies list to the Applied policies list.
 
-== Testing Policy Applications
+// == Testing Policy Applications
 
-To test the execution of policies, follow the steps in this section to apply a policy and use curl, for example `curl` `+http://localhost:8081 -vvv+` to test the policies.
+// To test the execution of policies, follow the steps in this section to apply a policy and use curl, for example `curl` `+http://localhost:8081 -vvv+` to test the policies.
```
Comment (kbostrom) [inline modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]:
```suggestion
// To test policy execution, follow these steps to apply a policy, and use curl (`curl` `+http://localhost:8081 -vvv+`) to test it.
```

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc [hunk 37]
```diff
@@ -136,43 +125,43 @@ You can use wildcards to remove properties that have similar names. For example,
...
 
-=== Add Request Header Policy
+// === Add Request Header Policy
 
-The procedure for testing the add or remove request header policies assume that you created a mule application that logs the headers received by the backend service.
+// The procedure for testing the add or remove request header policies assume that you created a mule application that logs the headers received by the backend service.
```
Comment (kbostrom) [inline modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]:
```suggestion
// The procedure for testing the add or remove request header policies assumes that you created a Mule application that logs the headers received by the backend service.
```

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc [hunk 38]
```diff
@@ -136,43 +125,43 @@ You can use wildcards to remove properties that have similar names. For example,
...
 
-. Apply the add request header policy to add a header as described in the previous section.
-. Run curl.
-. Check the log of the backend service to verify that request includes the additional header name and value you configured in step 1.
+// . Apply the add request header policy to add a header as described in the previous section.
```
Comment (kbostrom) [inline modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]: caps

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc [hunk 39]
```diff
@@ -136,43 +125,43 @@ You can use wildcards to remove properties that have similar names. For example,
...
+// . Run curl.
+// . Check the log of the backend service to verify that request includes the additional header name and value you configured in step 1.
```
Comment (kbostrom) [inline modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]:
```suggestion
// . Check the backend service log to verify that the request includes the additional header name and value you configured in Step 1.
```

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc [hunk 40]
```diff
@@ -136,43 +125,43 @@ You can use wildcards to remove properties that have similar names. For example,
...
 
 
-=== Remove Request Header Policy
+// === Remove Request Header Policy
 
-The procedure for testing the remove request header policy assumes that you created a mule application that logs the headers received by the backend service.
+// The procedure for testing the remove request header policy assumes that you created a mule application that logs the headers received by the backend service.
```
Comment (kbostrom) [inline modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]:
```suggestion
// The procedure for testing the Remove Request Header policy assumes that you created a Mule application that logs the headers received by the backend service.
```

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc [hunk 41]
```diff
@@ -136,43 +125,43 @@ You can use wildcards to remove properties that have similar names. For example,
...
 
-. Apply the remove request header policy to remove a header as described in the previous section.
-. Run curl.
-. Verify that the backend service excludes the extra header by checking the log.
+// . Apply the remove request header policy to remove a header as described in the previous section.
```
Comment (kbostrom) [inline modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]: Please apply to these three steps the same edits as made in the previous topic.

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc (hunk 38 above)
Comment (kbostrom) [inline modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]:
```suggestion
// . Apply the Add Request Header policy.
```

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc [hunk 42]
```diff
@@ -136,43 +125,43 @@ You can use wildcards to remove properties that have similar names. For example,
...
+// . Verify that the specified header is received.
 
-=== Remove Response Header Policy
+// === Remove Response Header Policy
 
-. Run curl.
-. Check the response to see which headers are being received by the client.
-. Apply the remove response header policy to remove a response from the header.
-. Run curl again.
-. Verify that the specified header is removed.
```
Comment (kbostrom) [inline modules/ROOT/pages/policy-mule3-add-remove-headers.adoc:175]: Ditto previous edits to the similar steps above

**Review by kbostrom — APPROVED** [review]
Complete quick edit

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc (hunk 7 above)
Comment (priyaayyar) [reply modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]: Yes, I went with adding. Thanks!

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc (hunk 8 above)
Comment (priyaayyar) [reply modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]: Ok! Thanks!

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc (hunk 9 above)
Comment (priyaayyar) [reply modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]: This seems already fixed! Could not find the sentence.

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc (hunk 10 above)
Comment (priyaayyar) [reply modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]: Done! Thanks!

Comment (priyaayyar) [reply modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]: "The prerequisites for using the header policies are:" Now added a heading too.

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc (hunk 18 above)
Comment (priyaayyar) [reply modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]: Done! Thanks!

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc (hunk 19 above)
Comment (priyaayyar) [reply modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]: Done! Thanks!

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc (hunk 20 above)
Comment (priyaayyar) [reply modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]: Done! Thanks!

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc (hunk 21 above)
Comment (priyaayyar) [reply modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]: Done! Thanks!

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc (hunk 22 above)
Comment (priyaayyar) [reply modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]: Done!

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc (hunk 23 above)
Comment (priyaayyar) [reply modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]: Done! Thanks!

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc (hunk 24 above)
Comment (priyaayyar) [reply modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]: Ok! Done!

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc (hunk 25 above)
Comment (priyaayyar) [reply modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]: Done! Thanks!

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc (hunk 26 above)
Comment (priyaayyar) [reply modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]: Text removed. Thanks!

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc (hunk 27 above)
Comment (priyaayyar) [reply modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]: Done!

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc (hunk 28 above)
Comment (priyaayyar) [reply modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]: Removed! Thanks!

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc (hunk 29 above)
Comment (priyaayyar) [reply modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]: Done! Thanks!

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc (hunk 39 above)
Comment (priyaayyar) [reply modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]: Removed text! Thanks!

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc (hunk 40 above)
Comment (priyaayyar) [reply modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]: Text removed! Thanks!

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc (hunk 41 above)
Comment (priyaayyar) [reply modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]: Text removed!

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc (hunk 38 above)
Comment (priyaayyar) [reply modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]: Text removed! Thanks!

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc (hunk 42 above)
Comment (priyaayyar) [reply modules/ROOT/pages/policy-mule3-add-remove-headers.adoc:175]: Text removed! Thanks!

File: modules/ROOT/pages/policy-mule3-add-headers-policy.adoc [hunk 43]
```diff
@@ -0,0 +1,112 @@
...
+
+== Applying the Header Injection Policy
+
+Perform the following steps to apply the policy:
+
+. From your left navigation, select *API Administration*.
+. Select the version to which you want to apply the policy.
+. From your left navigation, select *Policies*.
++
+The policies details page appears. If the header was already provided by the client or another policy, the value is overwritten with your header.
```
Comment (JaatadiaMulesoft) [inline modules/ROOT/pages/policy-mule3-add-headers-policy.adoc]: no

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc (hunk 8 above)
Comment (JaatadiaMulesoft) [inline modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]: Header Injection policy

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc [hunk 44]
```diff
@@ -4,79 +4,83 @@ include::_attributes.adoc[]
...
-+
-. Unzip each policy archive.
+For example, in the a request with header "foo=value1", if we add the header "foo=value2" with the policy, the result will be a request with header "foo=value1, value2"
 
-An XML and YAML file appears in each of the folders created by unzipping the archives. For example, the add-request-header-policy folder contains the following files:
+== Remove Request and Remove Response Headers
 
-* add-request.xml
-* add-request.yaml
+The Remove Request Header policy prevents receipt of one or more specified headers sent from the client to the backend service. The Remove Response Header policy prevents inclusion of one or more specified headers in a response from the backend service to the client.
```
Comment (JaatadiaMulesoft) [inline modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]: Header Removal policy

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc [hunk 45]
```diff
@@ -4,79 +4,83 @@ include::_attributes.adoc[]
...
+//. In Policy Configuration, choose the XML file, for example, `add-request.xml`, that you downloaded.
+//. Click Add.
+//+
+//image::gw-add-custom-pol.png[]
+//+
+//The policy, add request header policy in this example, appears on the Custom Policies page.
+//+
+//image::gw-custom-pol-list.png[]
+
+=== Applying the Header Injection Policies
```
Comment (JaatadiaMulesoft) [inline modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]: "policy"

File: modules/ROOT/pages/policy-mule3-add-remove-headers.adoc (hunk 24 above)
Comment (JaatadiaMulesoft) [inline modules/ROOT/pages/policy-mule3-add-remove-headers.adoc]: Header Removal Policy

**Review by JaatadiaMulesoft — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/policy-mule3-add-headers-policy.adoc (hunk 43 above)
Comment (priyaayyar) [reply modules/ROOT/pages/policy-mule3-add-headers-policy.adoc]: I assume you meant that the value is not overwritten by the header. I removed the sentence. A bit more detail about "no" for what would have helped.

**Review by JaatadiaMulesoft — CHANGES_REQUESTED** [review]
there is a duplicated page => modules/ROOT/pages/policy-mule3-remove-headers-policy.adoc is the same as  modules/ROOT/pages/policy-mule3-remove-headers.adoc

**Review by JaatadiaMulesoft — APPROVED** [review]
